flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/main/java/com/ververica/cdc/connectors/values/source/ValuesDataSourceOptions.java

```diff
@@ -26,20 +26,23 @@
 /** Configurations for {@link ValuesDataSource}. */
 public class ValuesDataSourceOptions {
 
-    public static final ConfigOption<ValuesDataSourceHelper.SourceEventType> SOURCE_EVENT_TYPE =
-            ConfigOptions.key("source.event.type")
-                    .enumType(ValuesDataSourceHelper.SourceEventType.class)
-                    .defaultValue(ValuesDataSourceHelper.SourceEventType.SINGLE_SPLIT_SINGLE_TABLE)
+    public static final ConfigOption<ValuesDataSourceHelper.EventSetId> EVENT_SET_ID =
+            ConfigOptions.key("eventSet.id")
```

Comment:
minor: follow flink option name style: `event-set.id`

Reply:
Address it.